internal/command/deploy/deploy_build.go

```diff
@@ -20,6 +20,30 @@ import (
 	"github.com/superfly/flyctl/terminal"
 )
 
+func multipleDockerfile(ctx context.Context, appConfig *appconfig.Config) error {
+	if len(appConfig.BuildStrategies()) == 0 {
+		// fly.toml doesn't know anything about building this image.
+		return nil
+	}
+
+	found := imgsrc.ResolveDockerfile(state.WorkingDirectory(ctx))
+	if found == "" {
+		// No Dockerfile in the directory.
+		return nil
+	}
+
+	config, _ := resolveDockerfilePath(ctx, appConfig)
+	if config == "" {
+		// No Dockerfile in fly.toml.
+		return nil
+	}
+
+	if found != config {
+		return fmt.Errorf("Use %s (in fly.toml) instead of %s", config, found)
```

Comment:
I'd like to keep the word "ignoring", it's impactful enough to make people pay attention to the warning.

```suggestion
		return fmt.Errorf("Ignoring %s, and using %s (from fly.toml).", found, config)
```